code/obj/machinery/plantpot.dm

```diff
@@ -1187,10 +1189,15 @@
 					seedcount++
 				getamount--
 
+			var/harvest_string = "You harvest [cropcount] item"
+			if (cropcount > 1)
+				harvest_string += "s"
 			if(seedcount)
-				boutput(user, "<span class='notice'>You harvested [cropcount] items and [seedcount] seeds.</span>")
-			else
-				boutput(user, "<span class='notice'>You harvested [cropcount] items.</span>")
+				harvest_string += " and [seedcount] seed"
+				if (seedcount > 1)
+					harvest_string += "s"
+			harvest_string += ".</span>"
```

Comment:
I think this last `</span` bit should be needed, as it's added in the `boutput` below?

Reply:
Will remove :)

Reply:
Resolved in latest commit.